File: java/get-content/GetItems.java
// DocSection: getting_content_get_items
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_ENVIRONMENT_ID>");

// Gets all content items
CompletionStage<ContentItemsListingResponse> listingResponse = client.getItems();
// EndDocSection

File: js/get-content/GetItems.js
// DocSection: getting_content_get_items
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  environmentId: '8d20758c-d74c-4f59-ae04-ee928c0816b7'
});

const response = await deliveryClient.items()
  .toPromise();
// EndDocSection